model/Image.php

```diff
@@ -233,10 +247,10 @@ public function SetRatioSize($width, $height) {
 		$heightRatio = $height / $this->height;
 		if( $widthRatio < $heightRatio ) {
 			// Target is higher aspect ratio than image, so check width
-			if($this->isWidth($width) && !Config::inst()->get('Image', 'force_resample')) return $this;
+			if($this->isWidth($width)) return $this;
```

Comment:
If its already changed for some reason then I withdraw my complaints.

This SemVer / 3.2 / 4.0 thing is getting really confusing - I feel like I have no idea sometimes and other times feel completely on top of it.

Anyway, changing the output of a public API method **is** a breaking change, regardless of what it changes and we should be careful not to make these decisions about the way things work without regarding even *potential* ways that they could have been used. Sure people probably aren't linking directly to that output, *but they might be* and should be able to until a non-BC version is made.